src/backend/assets/js/main.js

```diff
@@ -112,8 +112,49 @@ const loadVideoAndSlides = function(div, videoURL, slideURL) {
   }
 };
 
-window.handleClientLoad = handleClientLoad;
-window.initClient = initClient;
-window.handleAuthClick = handleAuthClick;
-window.listUpcomingEvents = listUpcomingEvents;
-window.loadVideoAndSlides = loadVideoAndSlides;
+$(document).ready(function () {
+  let filter;
+  //store room filters
+  let room ={
+    dateFilter:'.date-filter'
+  };
+  //store track filters
+  let track ={
+    dateFilter:'.date-filter'
+  };
+  //store schedule filters
+  let schedule={
+    dateFilter:'.day-filter'
+  };
```

Comment:
Shouldn't there be a space before and after every `=`?

I recommend getting an eslint plugin for your editor.

Reply:
But something else is wrong here.

The variables `room`, `track` and `schedule` are not used inside this function.

Reply:
@joeytwiddle please read schedule.hbs , room.hbs, track.hbs